src/main/g8/project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
-addSbtPlugin("com.mojolly.scalate" % "xsbt-scalate-generator" % "0.5.0")
```

Comment:
@takezoe Thanks for the pull request. It would be better to move https://github.com/skinny-framework/sbt-scalate-precompiler to scalate project. 

I have the admin permission of Scalate project now (at the time I've forked xsbt-scalate-generator, I was just a contributor of the project). So I can work on it. Wait a minute and please change the organization from `org.skinny-framework` to `org.scalatra.scalate`.